README.md

```diff
@@ -1,3 +1,24 @@
 # Indoor Navigation System [![Build Status](https://travis-ci.org/platisd/indoor-navigation-system.svg?branch=master)](https://travis-ci.org/platisd/indoor-navigation-system) [![codecov](https://codecov.io/gh/platisd/indoor-navigation-system/branch/dev/graph/badge.svg)](https://codecov.io/gh/platisd/indoor-navigation-system)
 
-[WIP] An indoor navigation system to guide users towards their colleagues' desks.
+[WIP] An indoor navigation system that guides you to the desk of your colleagues.
+
+## What?
+INS is a navigation system that utilizes a fleet of portable embedded devices, which have the ability to provide positioning information about themselves, to guide its user towards the desk of an employee or a colleague. The main scenario begins by searching for an employee on a mobile client. Then the system provides information regarding the whereabouts of the said employee's desk as well as navigational instructions on how to reach it.
```

Comment:
The embedded device themselves do not provide positioning information, but the rssi & AP info, right?. That information will only be determined on the backend and propagated towards the client applications.

Reply:
Yeah, I thought that this was made clear (therefore they just provide positioning information and cannot position themselves) but apparently it's not. What about this instead:

>..... a fleet of portable embedded devices, **the location of which can be calculated**, to guide its user towards...

Reply:
Or even simpler: "which can be located"

Reply:
Yeah, that should work. Good job.